Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk:latest
+FROM adoptopenjdk:15-jre
```

Comment:
I'm going to combine the "multiarch-15" discussion with my plans today/tomorrow to switch master/latest to be Java 11 based. In the process of that I have introduce a branch/image-tag `java8`. I have since regretted the "adopt" naming scheme as being too vendor specific, even though I'm loving their offering.

So...I'm going to introduce a `java15` branch/image-tag, use the base image you propose here, and just slide into a multi-architecture offering with java 15 and beyond.